### PR TITLE
feat(patterns/pagination): add pagination documentation

### PR DESCRIPTION
**Pull request by caqueste**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Comment by tiloyi** [comment]
Deploy URL : https://id679pagination-dot-demo-dot-mozaic-design-system-dtrp.ey.r.appspot.com

File: src/docs/Components/Pagination/index.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  This variation is not available yet.
+</Highlight>
+
+This variation is used on media pages like articles list pages.
+
+--- Preview ---
+
+**Vertcial pagination give you theses following informations:**
+
+- An indicator to show you the number of article you've seen, and how many articles are left.
```
Comment (tiloyi) [inline src/docs/Components/Pagination/index.mdx:38]: If the vertical pagination is not ready, is this text necessary?

File: src/docs/Components/Pagination/index.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: 'Pagination'
+order: 8
+description: "The pagination enable users to navigate from a page to another one, especially when you have to split a large content in several parts. It's mainly used on pages like the search engine result page or the list pages."
```
Comment (nherwegh) [inline src/docs/Components/Pagination/index.mdx:4]: Pagination enables users to navigate from a page to another one, especially when a large content needs to be splitted in several parts. It's mainly used on pages like the search engine result page or the list pages.

File: src/docs/Components/Pagination/index.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+status:
+  sketch: 'bêta'
+  scss: 'bêta'
+---
+
+> The pagination enable users to navigate from a page to another one, especially when you have to split a large content in several parts. It's mainly used on pages like the search engine result page or the list pages.
```
Comment (nherwegh) [inline src/docs/Components/Pagination/index.mdx:10]: Pagination enables users to navigate from a page to another one, especially when a large content needs to be splitted in several parts. It's mainly used on pages like the search engine result page or the list pages.

File: src/docs/Components/Pagination/index.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Variations
+
+### Horizontal navigation
+
+There is two mechanics in this case. You can use the arrows to go on previous or next pages, or you can used the select to go on the wanted page.
```
Comment (nherwegh) [inline src/docs/Components/Pagination/index.mdx:16]: There **ARE** …

File: src/docs/Components/Pagination/index.mdx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+--- Preview ---
+
+**Horizontal pagination give you theses following informations:**
```
Comment (nherwegh) [inline src/docs/Components/Pagination/index.mdx:20]: **Horizontal pagination giveS you the following informations:**

File: src/docs/Components/Pagination/index.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- A button to navigate to the previous page
+- A button to navigate to the next page.
+
+### Vertical navigation
+
+<Highlight theme="warning" title="Work in progress">
+  This variation is not available yet.
+</Highlight>
+
+This variation is used on media pages like articles list pages.
```
Comment (nherwegh) [inline src/docs/Components/Pagination/index.mdx:32]: (…) article list pages.

**Pull request merged**